<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>679</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolBar
{
	background-color: rgba(0, 0, 0,0.8);
}
QToolBar::separator
{
	background-color: rgba(255, 255, 255,0.2);
	width:2px;height:11;
}
QMenuBar{
	background-color: rgba(0, 0, 0,0.8);
	font:  16pt  &quot;文悦新青年体 W8&quot; ;
    color:white ;

spacing:10;}

QMenuBar::item{background-color:rgba(255, 255, 255,0);color:white ;}
QMenu{background-color:rgba(255, 255, 255,0.2);color:white ;}
QMenu::item{background-color:rgba(255, 255, 255,0.4);color:white ;}
QMenu::item:selected{background-color:rgba(255, 255, 255,0.6);color:black ;}
QStatusBar
{
background-color: rgba(255, 255, 255,0.2);
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>191</green>
       <blue>28</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QLabel" name="bgpic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-90</y>
      <width>1081</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="myImage.qrc">:/myImage/images/bg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bgwhite">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1091</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(255, 255, 255,0.35);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>bgpic</zorder>
   <zorder>bgwhite</zorder>
   <zorder>mdiArea</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>39</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>50</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>文悦新青年体 W8</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>38</width>
     <height>38</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="boldAct"/>
   <addaction name="italicAct"/>
   <addaction name="underlineAct"/>
   <addaction name="action_yangshi"/>
   <addaction name="actioncolor"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>33</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>文悦新青年体 W8</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/open.png</normaloff>:/myImage/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)...</string>
   </property>
   <property name="toolTip">
    <string>打开文件(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/save.png</normaloff>:/myImage/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/saveAs.png</normaloff>:/myImage/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/exit.png</normaloff>:/myImage/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/undo.png</normaloff>:/myImage/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/redo.png</normaloff>:/myImage/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>恢复(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/cut.png</normaloff>:/myImage/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/copy.png</normaloff>:/myImage/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/paste.png</normaloff>:/myImage/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭(&amp;O)</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>关闭所有窗口(&amp;A)</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>平铺(&amp;T)</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>层叠(&amp;C)</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/next.png</normaloff>:/myImage/images/next.png</iconset>
   </property>
   <property name="text">
    <string>下一个(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/previous.png</normaloff>:/myImage/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>前一个(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Backtab</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/about.png</normaloff>:/myImage/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="boldAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/bold.png</normaloff>:/myImage/images/bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗</string>
   </property>
  </action>
  <action name="italicAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/Italic.png</normaloff>:/myImage/images/Italic.png</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
  </action>
  <action name="underlineAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/under-line.png</normaloff>:/myImage/images/under-line.png</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
  </action>
  <action name="action_yangshi">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/language.png</normaloff>:/myImage/images/language.png</iconset>
   </property>
   <property name="text">
    <string>样式</string>
   </property>
  </action>
  <action name="actioncolor">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/product.png</normaloff>:/myImage/images/product.png</iconset>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/images/add.png</normaloff>:/myImage/images/add.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建(N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myImage.qrc"/>
 </resources>
 <connections/>
</ui>
